packages/docs/docs/medications/medication-codes.md

```diff
@@ -0,0 +1,87 @@
+---
+keywords:
+  - rxnorm
+  - ndc
+  - drugs
+  - drug codes
+---
+
+# Medication Code Systems
+
+## Introduction
+
+Medication codes play a fundamental role in crafting prescriptions, medication orders, and formularies. EHRs use standardized codes to describe medications to remove ambiguity, ensure patient safety, and streamline billing and analytics.
+
+In an ideal world, a universal standard of codes would be agreed upon to represent drugs. However, the reality is that there are multiple code systems, each with its unique characteristics and applications.
+
+This guide is dedicated to discussing two of the most significant, non-proprietary code systems in U.S. healthcare: [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) and [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory).
+
+- [RXNorm](https://www.nlm.nih.gov/research/umls/rxnorm/overview.html) is an international standard endorsed by standards-compliant bodies such as the US Core, NIH, and CDC.
+- [NDC](https://www.fda.gov/drugs/drug-approvals-and-databases/national-drug-code-directory) (National Drug Code) is a U.S. specific system administered by the Food and Drug Administration (FDA)
+
+While there are other proprietary drug databases with their proprietary code systems, they will not be explored in this guide.
+
+## RxNorm
+
+RxNorm is the preferred code system of the [U.S. Core](/docs/fhir-datastore/understanding-uscdi-dataclasses) and the National Institutes of Health (NIH), as well as most FHIR profiles.
```

Comment:
"preferred system of the U.S. Core" - reads weird to me

Maybe "preferred system of the US Core Medication Profile" ?

Also, reading the US Core docs, it looks like they do not use periods in "US" vs "U.S."